components-core/src/main/java/com/adyen/checkout/components/core/internal/provider/ActionComponentProvider.kt

```diff
@@ -15,12 +15,15 @@ import androidx.lifecycle.LifecycleOwner
 import androidx.lifecycle.SavedStateHandle
 import androidx.lifecycle.ViewModelStoreOwner
 import androidx.savedstate.SavedStateRegistryOwner
-import com.adyen.checkout.components.base.ActionComponentCallback
-import com.adyen.checkout.components.base.ActionDelegate
-import com.adyen.checkout.components.base.Configuration
-import com.adyen.checkout.components.model.payments.response.Action
+import com.adyen.checkout.components.core.action.Action
+import com.adyen.checkout.components.core.internal.ActionComponent
+import com.adyen.checkout.components.core.internal.ActionComponentCallback
+import com.adyen.checkout.components.core.internal.Component
+import com.adyen.checkout.components.core.internal.Configuration
+import com.adyen.checkout.components.core.internal.ui.ActionDelegate
 
 // TODO SESSIONS: DOCS
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
I guess we don't need `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` on `getDelegate()` method anymore after adding this to the whole interface

Reply:
I'd like to keep it there to make extra clear that it shouldn't be used